framework/decode/vulkan_object_info.h

```diff
@@ -461,6 +473,9 @@ struct PipelineInfo : public VulkanObjectInfoAsync<VkPipeline>
 
     // Is VK_DYNAMIC_STATE_VERTEX_INPUT_BINDING_STRIDE_EXT enabled
     bool dynamic_vertex_binding_stride{ false };
+
+    // map capture- to replay-time shader-group-handles
+    std::unordered_map<graphics::shader_group_handle_t, graphics::shader_group_handle_t> shader_group_handle_map;
```

Comment:
@NickDriscoll  this LUT is pretty much the outcome of this PR. 
we'll be able to map from capture- to replay-time handles (but have to figure out how/when/where we want to do it ... in a follow-up)

Reply:
Makes sense to me.